tools/webdriver/webdriver/bidi/modules/network.py

```diff
@@ -57,11 +57,87 @@ def _add_intercept(self, result: Mapping[str, Any]) -> Any:
         assert result["intercept"] is not None
         return result["intercept"]
 
+    @command
+    def continue_with_auth(
+        self,
+        request: str,
+        action: str,
+        credentials: Optional[Dict[str, str]] = None
+    ) -> Mapping[str, Any]:
+        params: MutableMapping[str, Any] = {
+            "request": request,
+            "action": action,
+        }
+
+        if action == "provideCredentials" and credentials is not None:
+            params["credentials"] = {}
+            if "type" in credentials:
+                params["credentials"]["type"] = credentials["type"]
+            if "username" in credentials:
+                params["credentials"]["username"] = credentials["username"]
+            if "password" in credentials:
+                params["credentials"]["password"] = credentials["password"]
+
+        return params
+
+    @command
+    def continue_request(self,
+                         request: str,
+                         method: Optional[str] = None,
+                         url: Optional[str] = None) -> Mapping[str, Any]:
+        params: MutableMapping[str, Any] = {
+            "request": request,
+        }
+
+        if method is not None:
+            params["method"] = method
+
+        if url is not None:
+            params["url"] = url
+
```

Comment:
It would be nice to have a comment added to all the network interception commands that some of the optional arguments need to be added. Maybe we could even name them directly?

Reply:
Done